src/SIL.XForge.Scripture/ClientApp/src/app/shared/notice/notice.component.html

```diff
@@ -1,4 +1,7 @@
-<div [ngClass]="{ outline }" [class]="type">
-  <mat-icon *ngIf="icon">{{ icon }}</mat-icon>
-  <span><ng-content></ng-content></span>
-</div>
+<ng-container *transloco="let t; read: 'notice'">
+  <div [ngClass]="{ outline }" [class]="type">
+    <mat-icon *ngIf="icon">{{ icon }}</mat-icon>
+    <span class="notice-content"><ng-content></ng-content></span>
+    <a mat-flat-button *ngIf="helpLink" [href]="helpLink" target="_blank">{{ t("learn_more") }}</a>
```

Comment:
How about adding this to the ExternalUrlService:
``` ts
get rolesHelpPage(): string {
  return this.helps + '/Concepts/Roles.htm';
}
```
And then referencing it here.

Keep in mind #1823 though; it's making needed changes to the way we link to helps, which may impact this.

Then we can get rid of `@Input() helpLink?: string;`. I don't think the parent component should be responsible for figuring out where the roles help page is at.